unofficial/c511027102.lua

```diff
@@ -0,0 +1,102 @@
+--ZW－荒鷲激神爪
+--ZW - Eagle Claw (anime)
+--Script by Rundas
+local s,id=GetID()
+function s.initial_effect(c)
+	--normal summon without tribute
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(id,0))
+	e1:SetProperty(EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_UNCOPYABLE)
+	e1:SetType(EFFECT_TYPE_SINGLE)
+	e1:SetCode(EFFECT_SUMMON_PROC)

```

Comment:
This is slightly off-shape, my advice is go check out how Cyber Shark is doing with Normal Summon without tributing.